### ISSUE TITLE
Support for ghc-8.10.x?

### ISSUE THREAD

**Issue by peti** [reflex-frp/reflex#443]
I've tried to compile reflex-0.7.1.0 with ghc-8.10.2, but I've run into the following compiler error:

~~~
src/Reflex/Spider/Internal.hs:2742:21: error:
    Not in scope: type variable ‘k2’
     |
2742 |     :: forall (k :: k2 -> *) q (v :: k2 -> *). GCompare k
     |                     ^^

src/Reflex/Spider/Internal.hs:2742:38: error:
    Not in scope: type variable ‘k2’
    Perhaps you meant ‘k’ (line 2742)
     |
2742 |     :: forall (k :: k2 -> *) q (v :: k2 -> *). GCompare k
     |                                      ^^
[39 of 44] Compiling Reflex.Profiled  ( src/Reflex/Profiled.hs, dist/build/Reflex/Profiled.o, dist/build/Reflex/Profiled.dyn_o )

src/Reflex/Profiled.hs:149:26: error:
    Not in scope: type variable ‘z’
    |
149 |   mergeG :: forall (k :: z -> *) q v. GCompare k
    |        
~~~

Is there by any chance an easy way to fix that?

**Comment by ProofOfKeags** [comment]
Just ran into this as well. My guess is that quantifying k2 would fix it. I was hoping to be able to turn on some extensions and have things just work but no luck.

**Comment by alexfmpe** [comment]
#444

**Comment by ali-abrar** [comment]
Fixed in version 0.8.1.0
